app/components/workflow-toggle.spec.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import assert from 'assert';
+import { shallow } from 'enzyme';
+import sinon from 'sinon';
+import WorkflowToggle from './workflow-toggle';
+
+const project = {
+  display_name: 'test'
+};
+
+const workflow = {
+  display_name: 'test'
+};
+
+describe('WorkflowToggle', () => {
+  let wrapper;
+  let handleToggleActiveStub;
+
+  before(() => {
+    handleToggleActiveStub = sinon.stub(WorkflowToggle.prototype, 'handleToggleActive');
+    wrapper = shallow(<WorkflowToggle project={project} workflow={workflow} field="active" />);
+    handleToggleActiveStub.restore();
```

Comment:
Calling restore here shouldn't be necessary. You haven't done anything with the stub yet.